app/views/supers/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="container">
+  <h1> <%= @super.name %></h1>
+
+    <p><%= @super.location %></p>
```

Comment:
All looks good! The only thing is lines 4-9 look like they are indented one space too much